src/components/hallOfFame.html

```diff
@@ -4,26 +4,23 @@
    <div class="modal-content">
        <div class="modal-header">
            <div style="width: 100%;">
-               <img src=""
-                    data-bind="attr:{ src: 'assets/images/oak.png'}">
+               <img src="assets/images/oak.png">
                <h5 class="modal-title" id="hallOfFameModalLabel">Professor Oak</h5>
-
            </div>
        </div>
        <div class="modal-body">
            <p>
-               So, you won! Congratulations! You're the new Pokémon League champion! You've grown
-               up so
-               much since you first left with <span
-                   data-bind="text: GameConstants.Starter[player.starter]">starter</span>! You have
-               worked
-               hard to become the new League Champion!
+               So, you won! Congratulations!<br/>
+               You're the new Pokémon League champion!<br/>
+               You've grown up so much since you first left with <span data-bind="text: GameConstants.Starter[player.starter]">starter</span>!<br/>
```

Comment:
```suggestion
               You have grown up so much since you first left with <span data-bind="text: GameConstants.Starter[player.starter]">starter</span>!<br/>
```